release-notes/unreleased/IGAPP-363-hide-dashboard.yml

```diff
@@ -0,0 +1,6 @@
+issue_key: IGAPP-363
+show_in_stores: false
```

Comment:
I think this should go into the stores as it is a nice feature for cities and users